src/renderer/Core/I18n/getCoreResources.ts

```diff
@@ -55,6 +55,8 @@ export const getCoreResources = (): { namespace: string; resources: Resources<Tr
                     searchHistoryEnabledDescription: "The search history lets you see your previous searches",
                     searchHistoryLimit: "Search History Limit",
                     searchHistoryLimitHint: "Limits the maximum number of search history items",
+                    waylandWarning: "Wayland detected. Hotkeys may not work.",
```

Comment:
Can you quickly explain what "wayland" exactly is? 😅 Why dont' we need the hotkey in this case?

Reply:
Wayland is a new window management system many Linux distros are switching to. Due to security, windows are no longer able to read keyboard inputs when not in focus, and thus, no way to listen for global hotkeys. There's a system API in the works to fix this (that Chromium merged last week), but it is still under development.